README.md

```diff
@@ -2,6 +2,17 @@
 
 This adds [Julia language](https://julialang.org/) support to [CTAGS](https://ctags.io/).
 
+To install run
+```julialang
+using Pkg
+Pkg.add("Interpolations")
+```
+or
+```
+] add JuliaCtags
+```
+To obtain the ctags configuration file use `get_config()` which will create a file called `julia_ctags` in the current directory.
```

Comment:
How about we make:
```suggestion
To obtain the ctags configuration file use `write_config(dir=pwd())` which will create a file called `julia_ctags` in the targetted directory.
```

Reply:
How about we split the difference? `write_config(filename="./julia_ctags")`

Reply:
sounds great to me